index.html

```diff
@@ -10,6 +10,7 @@
 			<div class="undo" id="undo">
 				<p>UNDO</p>
 			</div>
+			<div id = "turnIndicator" >Reds turn</div>
```

Comment:
I think you need to add `style = "color:red"` here so that the turn is known for the first time also.